sdk/ai/azure-ai-projects/samples/agents/sample_agents_basics_with_tracing.py

```diff
@@ -0,0 +1,84 @@
+# ------------------------------------
```

Comment:
I have samples for both console and Azure Monitor tracing, each with unique names. See the xxx_traces.py samples here: https://github.com/Azure/azure-sdk-for-python/tree/dargilco/azure-ai-client/sdk/ai/azure-ai-projects/samples/inference . It would be good to have two such examples for Agents as well.

Reply:
You can do that in a separate PR, after I complete my next PR (I need to rename ".diagnostics" to ".telemetry")

Reply:
changed the samples to follow same pattern